### PR TITLE
Update min symfony version to ^7.0.0

### PR DESCRIPTION
**Pull request by dmytrome**
### What:

- [x] Bug Fix
- [x] New Feature

### Description:

Update min Symfony version to ^7.0.0 to support latest Symfony release.

### Related:

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -15,21 +15,21 @@
...
+        "php": "^8.2.0",
         "nyholm/psr7": "^1.8.1",
         "openai-php/client": "^0.8.0",
         "psr/http-client": "^1.0.3",
         "psr/http-factory": "^1.0.2",
-        "symfony/config": "^5.4.21|^6.3.8",
-        "symfony/dependency-injection": "^5.4.21|^6.3.8",
-        "symfony/http-client": "^5.4.21|^6.3.8",
-        "symfony/http-kernel": "^5.4.21|^6.3.8"
+        "symfony/config": "^7.0.0",
```
Comment (GromNaN) [inline composer.json]: There is no reason to change the minimum values.
`^5.4|^6.3|^7.0` should be the used as all this versions are compatibles.

**Review by GromNaN — APPROVED** [review]

File: composer.json [hunk 2]
```diff
@@ -15,21 +15,21 @@
         }
     ],
     "require": {
-        "php": "^8.1.0",
+        "php": "^8.2.0",
```
Comment (GromNaN) [inline composer.json:18]: Please revert this also.

```suggestion
        "php": "^8.1",
```

Comment (dmytrome) [reply composer.json:18]: It won't work
image

File: .github/workflows/tests.yml [hunk 3]
```diff
@@ -9,8 +9,8 @@ jobs:
       fail-fast: true
       matrix:
         os: [ubuntu-latest]
-        php: [8.1, 8.2]
-        symfony: [5.4.*, 6.2.*]
+        php: [8.2]
```
[the remark this answers is not in the document]
Comment (dmytrome) [reply .github/workflows/tests.yml:12]: Minimal version for Symfony 7.0 is 8.2

[the remark this answers is not in the document]
Comment (GromNaN) [reply .github/workflows/tests.yml:12]: You are right. You can add extra matrix values with `includes`:

```yaml
include:
  - php: "8.1"
    symfony: "5.4.*"
  - php: "8.1"
    symfony: "6.3.*"
  - php: "8.1"
    symfony: "6.4.*"
```

File: composer.json (hunk 2 above)
Comment (GromNaN) [reply composer.json:18]: This is the minimum version of PHP to install this dependency. If you have a project with PHP 8.2, it satisfies the contraint on PHP 8.1.

**Review by GromNaN — APPROVED** [review]
Thank you @dmytrome

**Pull request merged**